internals/core_models.py

```diff
@@ -116,6 +116,7 @@ class FeatureEntry(ndb.Model):  # Copy from Feature
   finch_name = ndb.StringProperty()
   non_finch_justification = ndb.TextProperty()
   ongoing_constraints = ndb.TextProperty()
+  rollout_plan = ndb.IntegerProperty(required=True, default=ROLLOUT_100)
```

Comment:
I imagine this is the case, but hopefully this will now assume that all previous FeatureEntry entities have a value here and this will not require backfilling.

Reply:
Correct, the default value is used when an existing stored lacks that field.